Gemfile

```diff
@@ -69,6 +69,11 @@ gem 'rack-cors'
 # Use Redis adapter to run Action Cable in production
 # gem "redis", ">= 4.0.1"
 
+# Sentry integration according to their documentation [https://docs.sentry.io/platforms/ruby/guides/rails/]
+gem "stackprof"
```

Comment:
`stackprof` is only required if we enable profiling and/or tracing. I think for now we should leave both off and remove this gem.